tex/alg-NT/frobenius.tex

```diff
@@ -599,8 +599,18 @@ \section{Frobenius elements control factorization}
 Once again we have a partition of $n$;
 we call this the \vocab{factorization pattern} of $f$ modulo $p$.
 So, to see the factorization pattern of an unramified $p$ in $\OO_E$,
-we just have to know the factorization pattern of the $f \pmod p$.
+we just have to know the factorization pattern of the $f \pmod p$.\\
```

Comment:
Don't use `\\` here, just hit newline twice to start a new paragraph. Ditto comments for everything else that follows.